### PR TITLE
Add getGender()

### PR DESCRIPTION
**Pull request by sbimochan**

### CODE REVIEW

File: src/Contracts/User.php [hunk 1]
```diff
@@ -38,4 +38,11 @@ public function getEmail();
      * @return string
      */
     public function getAvatar();
+
+    /**
+     * Get the gender of the user.
+     *
+     * @return string
+     */
+    public function getGender();
```
Comment (crynobone) [inline src/Contracts/User.php]: Is this targeting next major release? cause this potentially a huge breaking change.

Comment (sbimochan) [inline src/Contracts/User.php]: Absolutely not. I just edited socialite in project to pull gender and thought other people may need this function too.

Comment (GrahamCampbell) [inline src/Contracts/User.php]: > Absolutely not.

It needs to, because adding a new method to an interface is a major breaking change.

**Comment by taylorotwell** [comment]
Breaking change.

**Pull request closed without merging**